src/engraving/rw/read114/read114.cpp

```diff
@@ -2835,7 +2835,9 @@ muse::Ret Read114::readScore(Score* score, XmlReader& e, ReadInOutData* out)
             e.skipCurrentElement();
         } else if (tag == "page-layout") {
             compat::PageFormat pf;
```

Comment:
Looks like this `PageFormat` is being discarded too. Perhaps we can do something better with that as well? (Perhaps with help of some methods in `pageformat.cpp`, by exposing those via `pageformat.h`?)

Reply:
Hmpf, why didn't I see that? Probably because I'm overwriting those with my style file anyway...
That too is a regression vs. 2.x
Will take a look...

Reply:
2.x reads them right, and saves them right, so 3.x and 4.x do read them right too, so we'd 'just' need to get the 1,x scores to get their page format being read by the 2.x routines

Reply:
Seems I could fix it for my 3.7 Evolution (see https://github.com/Jojo-Schmitz/MuseScore/pull/873), but can't get it to work for upstream master

Reply:
Maybe the page format issue can be deferred to a different PR?

Reply:
It could, but I wonder why the same code as for 3.x can't work in master. As far as I can see, you just need to expose `initPageFormat` etc. via the `pageformat.h` header. Or is that not the problem?

Reply:
Apparently not.
First a plain rebase of the working part, then I'll push my attempt on the rest, it just won't build

Reply:
Hold on, I've just been too stupid...

Reply:
Got it working now. Previously I left `initPageFormat()` and `setPageFormat()` inside the `class PageFormat` in pagefomat.h, and just replaced `static ` with `PageFormat` in pageformat.cpp, this just didn't build

Reply:
I do expect a bunch of unit test failures, basically for all the `read114` tests

Reply:
Fixed the unit tests.

Interesting little bug (none to fix here IMHO): while importing `pageTwosided` from the 1.x scores works just fine, the corresponding setting in the 4.x ref.mscx files seem to get ignored!?! Check style_ref.mscx and text_styles.mscx (they have it set), compare with the import of style.mscx and textstyles.mscx (they have it unset). No idea what might be causeing this, maybe it is limited value of plain mscx **files** (vs. directories) in Mu4, i.e. is is taken from an mscz file's score_style.mss, but not from a plain .mscx?